Library/Formula/flif.rb

```diff
@@ -0,0 +1,24 @@
+class Flif < Formula
+  desc "Free Lossless Image Format"
+  homepage "http://flif.info"
+  url "https://github.com/FLIF-hub/FLIF/archive/v0.1-alpha.tar.gz"
+  version "0.1-alpha"
+  sha256 "b15229e07b43758bb993aa912a08d66f62e078e3f4964b13ffe7a13b6b3ecd07"
+
+  head "https://github.com/FLIF-hub/FLIF.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "libpng"
+
+  def install
+    system "make"
+
+    bin.mkpath
+    bin.install "flif"
+    include.install %w[flif.h flif_config.h]
+  end
+
+  test do
+    system bin/"flif", "--help"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!